src/Button.js

```diff
@@ -2,18 +2,25 @@
 import React from 'react';
 
 function resetProduct() {
```

Comment:
I can see what you're trying to do here, but you don't want to have functions in your button component. Functions should be in other components or in your index.js file and then called when the button is clicked. My suggestion would be to remove this function from the button component, you only want to render the button here and not have any logic.

In the comment below, I've added examples of how to render the button in the return statement with an `onClick` prop.

In your index.js file, you could create a simple function that shows an alert:

```
function sayHello() {
        alert('Hello')
}
```

Next, import the button as you have done at the top of the index.js file. Now, render the button component and call the function.

`<Button onClick={sayHello}>Hello Alert</Button>`

It's probably best to create an App component and import this into the index.js file. Here's the App component:

```
import React from 'react'
import Button from './Button/Button'

const App = () => {

    function sayHello() {
        alert('Hello')
    }

    return (
        <Button onClick={sayHello}>Hello Alert</Button>
    );
}

export default App
```

And the index.js file can be kept simple:

```
import React from 'react'
import { render } from 'react-dom'
import App from './App'

render(
    <App />,
    document.getElementById('root')
);
```

Reply:
Cheers @justtoconfirm  the first part of this has been actioned and re-commited as proof of concept I understood what you suggested. And it still works for me! I can now look at your second suggestion and create a separate App component, cheers

Reply:
> Cheers @justtoconfirm the first part of this has been actioned and re-commited as proof of concept I understood what you suggested. And it still works for me! I can now look at your second suggestion and create a separate App component, cheers

Cheers @justtoconfirm I have now moved the function out of the button component, created a new component for the function and imported this into the index.js file and it all still works